sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala

```diff
@@ -1333,4 +1334,19 @@ class ColumnarBatchSuite extends SparkFunSuite {
 
       column.close()
   }
+
+  testVector("WritableColumnVector.reserve(): requested capacity is too large", 1024, ByteType) {
+    column =>
+      val capacity = Integer.MAX_VALUE - 1
```

Comment:
@ala . Could you make it more clearly like the following because this test case aims to cover [the corner case of existing code](https://github.com/apache/spark/blob/master/sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java#L85-L86) between `MAX_CAPACITY + 1` and `Integer.MAX_VALUE`?

```scala
- val capacity = Integer.MAX_VALUE - 1
+ val capacity = column.MAX_CAPACITY + 1
```

Reply:
Good idea.

Reply:
Actually, we have an existing test coverage for this in this test suite; [test("exceeding maximum capacity should throw an error")](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala#L1209-L1226). Can we remove this test case from this PR?

Reply:
Ok.